tests/integration/helpers.py

```diff
@@ -38,7 +38,7 @@ async def scale(ops_test: OpsTest, app, units):
         status="active",
         idle_period=30,
         raise_on_blocked=True,
-        timeout=300,
+        timeout=600,
```

Comment:
Yup, I tried using `async with ops_test.fast_forward():` but that ended up breaking the scaling integration test, but perhaps this is because it is setting the interval to 10s which might be too fast for the health checks. Will try setting to 1m instead to see if that makes the integration test run faster without breaking it.